document.go

```diff
@@ -27,6 +27,9 @@ type Document interface {
 	NumPlainTextBytes() uint64
 
 	AddIDField()
+
+	SetStoredFieldsBytes(uint64)
+	GetStoredFieldsBytes() uint64
```

Comment:
Rename to StoredFieldsBytes().

Reply:
On second thought, you shouldn't need this method too, if you simply export the field.

Reply:
We might need the Get API, because the index.Document() API returns an interface Document, and we won't be able to access the StoredFieldsBytes field in the struct that's behind the interface